src/parser/core/modules/spells/bfa/azeritetraits/SynergisticGrowth.js

```diff
@@ -0,0 +1,21 @@
+import SPELLS from 'common/SPELLS';
+import GenericStatProc, { azeriteStats } from 'parser/core/modules/spells/bfa/azeritetraits/GenericStatProc';
+import STAT from 'parser/core/modules/features/STAT';
+
+export const STAT_TRACKER = {
+  mastery: combatant => azeriteStats(combatant.traitsBySpellId[SPELLS.BLOOD_RITE.id]).stat,
+};
+
+/**
+ * Synergistic Growth
+ * Gain x mastery while active
+ *
+ * Example report: /report/4qXRxW1fYLPaJgMA/7-Mythic+Zul+-+Wipe+1+(1:25)/8-Monkacarl
+ */
+class SynergisticGrowth extends GenericStatProc {
+  constructor(...args) {
+    super(SPELLS.SYNERGISTIC_GROWTH.id, SPELLS.SYNERGISTIC_GROWTH_BUFF.id, STAT.MASTERY, ...args);
+  }
+}
```

Comment:
As far as I remember these buffs are currently in a resto druid module from this PR https://github.com/WoWAnalyzer/WoWAnalyzer/pull/2235/files#diff-d4d8abcfdfba4098a816011f1545e4dd

Can we have them moved out

Reply:
Huh, I didn't realize they had added it. Should we check for other general traits that are implemented for specific classes?

Reply:
Wouldn't hurt to do a global search for the spell IDs when implementing, just to double check